src/components/Flipper/Flipper.tsx

```diff
@@ -208,6 +212,10 @@ function Flipper() {
                                     <Form.Label style={{ cursor: "pointer" }} onClick={onArrowRightClick}>To newest:</Form.Label>
                                     <span style={{ cursor: "pointer" }} onClick={onArrowRightClick}> <ArrowRightIcon /></span>
                                 </Form.Group>
+                                <Form.Group>
+                                    <Form.Label style={{ cursor: "pointer" }} onClick={onArrowRightClick}>Clear flips:</Form.Label>
```

Comment:
the onClick is binds to the wrong function.
I suggest you add the onClick onto the `Form.Group` element.